en/docs/installation.md

```diff
@@ -202,7 +202,7 @@ Type 'help' or 'h' for help. Type 'quit' or 'q' to quit console.
 =============================================================================================
 ```
 
-### Acquire information through console
+### Query blockchain's information
```

Comment:
Recommend to add `console`, eg. `Query blockchain's information through console`

Reply:
the whole chapter named `Using console`,  so I think that there is no need to add `through the console `